debezium-testing/mysql_repl.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: v1
```

Comment:
Does this file belong to this PR?

Reply:
NO idea doesn't.... that's a rogue k8 descriptor :)